build/util/src/lib.rs

```diff
@@ -48,38 +48,53 @@ pub fn expose_target_board() {
 /// reflect that by having its member (or members) be an `Option` type.
 ///
 pub fn config<T: DeserializeOwned>() -> Result<T> {
-    toml_from_env("HUBRIS_APP_CONFIG", Some("[config]"))
+    toml_from_env("HUBRIS_APP_CONFIG")?.ok_or_else(|| {
```

Comment:
Out of curiosity, why `?.ok_or_else(|| { anyhow!(...) })` instead of `.context(...)?`?

Reply:
Good point. Will fix.

Reply:
Ahhhhhh -- the `?` is load bearing.

This is converting a `Result<Option<T>>` into a `Result<T>` by (1) unwrapping the outer bits and then (2) applying [`Option::ok_or_else`](https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or_else) to the interior to convert `None` into an error.